src/components/organisms/administration/PrivacySettings.unit.js

```diff
@@ -35,11 +35,48 @@ describe("PrivacySettings", () => {
 	});
 
 	describe("env config", () => {
+		it("should enable student visibility switch", () => {
```

Comment:
I think the description should give a little more context, maybe like so:
```suggestion
		it("should enable student visibility switch according to env var", () => {
```

Reply:
...also in the other tests.

Reply:
those tests are already described as 'env-config' and I wanted to keep the naming 'convention' that was already in the file, let me know if you still want me to rename them

Reply:
I don't like that it reads just "env-config should enable student visibility switch" or "env-config should disable student visibility switch". There needs be a condition. Maybe better than "according to env var" would be "if env var for configurability is true"...

Reply:
I've restructured the tests in the file, please have a look again.